errs/errors.go

```diff
@@ -73,6 +78,8 @@ func WrapTyped(cause error) *Error {
 	if cause == nil {
 		return nil
 	}
+	// Intentionally not checking to see if there is a deeper wrapped *Error as the error must be wrapped again in order
+	// to avoid losing information and still return a *Error
 	if err, ok := cause.(*Error); ok { //nolint:errorlint // Explicitly only want to look at this exact error and not things wrapped inside it
 		return err
 	}
```

Comment:
Is the old code between new lines 58-85 actually needed now? Won't the code on lines 54-57 cause them to never trigger?

Reply:
Oh yeah, you're right, I'll remove it.